include/partitioning/parmetis_helper.h

```diff
@@ -26,6 +26,8 @@
 
 #ifdef LIBMESH_HAVE_PARMETIS
 
+#include "libmesh/id_types.h"
+
```

Comment:
This could be because we get Parmetis from PETSc in any builds where we run `make test_headers`?

Reply:
That might be it!  We even autodetect it and prefer it to our own if we find it, don't we?  So even my own `make test_headers` runs would have been fooled.